Description: [documentation] FAQ.html cleanup
 This patch removes the inclusion of the W3C validator icon and link, as well
 as other external images, which induced a privacy breach (fixes lintian error
 privacy-breach-w3c-valid-html and warning privacy-breach-generic). The
 internal links (anchors) were fixed.
Author: Matteo Cypriani <mcy@lm7.fr>
Last-Update: 2010-05-12
--- a/doc/FAQ.html
+++ b/doc/FAQ.html
@@ -11,20 +11,10 @@
   </head>
 
   <body bgcolor="#ffffff">
-<base href="http://fdutils.linux.lu">
 
 <div id="frame">
   <div id="logobar">
-
-  <table border=0 width="100%">
-   <tr>
-    <td rowspan=2 valign="top">
-     <A HREF="http://www.transfair.lu/petition/campcafe.html"><img src="http://alain.knaff.lu/BigNoise.jpg" border="0" alt="Join the Big Noise! Pay fair prices to coffee growers" width="239" height="145"></A>
-    <td align="left">
-     <center><a href="http://swpat.ffii.org/"><img src="images/banner09.png" alt="No Software Patents!" width=468 height=60></a></center><p>
-  <tr>
-    <td><h1 class="globaltitle">Fdutils</h1>
-  </table>
+    <h1 class="globaltitle">Fdutils</h1>
   </div>
 
   <div id="leftcolumn">
@@ -32,36 +22,18 @@
     <div id="navbar">
 
     <center><a href="http://alain.knaff.linux.lu">
-    <img src="images/home.gif" border=0 alt="[To my homepage]" width="31" height="24"></a></center>
+    [To my homepage]</a></center>
 
 
 <ul>
- <li><a href="index.html">Introduction</a>
- <li><a href="download.html">Downloads</a>
- <li><a href="mailing.html">Mailing list/doc</a>
- <li><a href="faq.html#start">Frequently asked questions</a>
- <li><a href="disk-id.html">How to identify<br>an unknown disk</a>
+ <li><a href="http://fdutils.linux.lu/index.html">Introduction</a>
+ <li><a href="http://fdutils.linux.lu/download.html">Downloads</a>
+ <li><a href="http://fdutils.linux.lu/mailing.html">Mailing list/doc</a>
+ <li><a href="http://fdutils.linux.lu/faq.html#start">Frequently asked questions</a>
+ <li><a href="http://fdutils.linux.lu/disk-id.html">How to identify<br>an unknown disk</a>
 </ul>
 
     </div> <!-- navbar -->
-    <!-- the Lilux penguin -->
-    <div id="penguin">
-	    <center><a class="picture" href="http://www.lilux.lu" target="_blank"><img src="images/lilux_01.gif" width="90" height="104" border="0" alt="lien vers http://www.lilux.lu"></a></center>
-	    <center><a class="picture" href="http://validator.w3.org/check/referer"><img border="0" src="http://www.w3.org/Icons/valid-html401" alt="Valid HTML 4.01!" height="31" width="88"></a></center>
-
-    </div> <!-- penguin -->
-
-<br>
-
-<center><a href="http://www.netcraft.com/cgi-bin/Survey/whats?host=fdutils.linux.lu">
-<img src ="images/powered.gif" width=90 height=53 border=0 alt="Powered By ...?">
-</a></center>
-
-<br>
-
-<center><A HREF="http://www.anybrowser.org/campaign/">
-<IMG SRC="images/viewed-globe.gif" ALT="Content Enhanced - Use Any Browser" BORDER="0" ALIGN="MIDDLE" WIDTH="147" HEIGHT="42">
-</A></center>
 
     </div> <!-- leftcolumn -->
     <div id="contentright">
@@ -72,7 +44,7 @@
 <div align=justify>
 
 Most of the floppy utilities mentioned in this FAQ can be found in the
-<a href="../fdutils/index.html">fdutils</a> package. For others,
+<a href="http://fdutils.linux.lu/">fdutils</a> package. For others,
 consult the lsm or archie. Alpha releases of fdutils are named
 fdutils-<em>version</em>-<em>YearMonthDay</em>..diff.gz
 
@@ -181,7 +153,7 @@
 executing the command, in order to find out which messages were
 generated by the command, and which ones were generated by earlyer
 commands. Alternatively, you may get the <a
-href="misc/user-printk.tgz">user-printk</a>
+href="http://fdutils.linux.lu/misc/user-printk.tgz">user-printk</a>
 module which allows you to insert "comments" into the stream of
 console messages.
 
